src/containerapp/HISTORY.rst

```diff
@@ -3,6 +3,10 @@
 Release History
 ===============
 
+0.3.1
+++++++
+* Update "az containerapp github-action add" parameters
```

Comment:
lets explicitly call out docker-path replaced by context-path

Reply:
Just fixed this